Page searchable toggle

kulturbande opened the pull request:
It is possible to remove pages from the index that have a disabled searchable - flag. This is the companion commit to the PR in Alchemy CMS.

Reference: https://github.com/AlchemyCMS/alchemy_cms/pull/2414

tvdeyen commented on Gemfile:
```diff
@@ -3,7 +3,7 @@ source "https://rubygems.org"
 
 gem "rails", "~> 6.0.0"
-ENV.fetch("ALCHEMY_BRANCH", "6.0-stable").tap do |branch|
+ENV.fetch("ALCHEMY_BRANCH", "6.1-stable").tap do |branch|
```
This change should be extracted into a separate PR that provides Alchemy 6.1 compatibility 

In that PR we also want to make sure to test both 6.0 and 6.1

tvdeyen commented on app/controller/alchemy/pg_search/controller_methods.rb:
```diff
@@ -70,11 +70,14 @@ def search_result_page
           raise "No searchresults page layout found. Please add page layout with `searchresults: true` into your `page_layouts.yml` file."
         end
-        page_layout
+        page_layouts.first
```
This change should be as well extracted into a separate PR that provides Alchemy 6.1 compatibility

kulturbande committed:
Page searchable toggle

It is possible to remove pages from the index that have a disabled searchable - flag. This is the companion commit to the PR in Alchemy CMS.

Reference: https://github.com/AlchemyCMS/alchemy_cms/pull/2414
